bitbybit/src/bitfield/mod.rs

```diff
@@ -556,3 +568,41 @@ fn const_name(field_name: &Ident, suffix: &str) -> Ident {
     syn::parse_str::<Ident>(&name)
         .unwrap_or_else(|_| panic!("bitfield!: Error creating {name} name"))
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_mask_underflow() {
+        assert_eq!(mask_for_width_and_offset(128, 0), u128::MAX);
+    }
+
+    #[test]
+    #[should_panic]
```

Comment:
should_panic should only have one test in it. Otherwise you risk a false negative if the first line panics

Reply:
good point. I'll split this up